### ISSUE TITLE
Create svg images

### ISSUE THREAD

**Issue by jwnigel** [jwnigel/permaculture#25]
We'll need svg images for different tree / shrub / plant forms. Obviously not all plants will have a unique image but a certain number of categories will. Think about the "habit" column in this[ EFG database](https://docs.google.com/spreadsheets/d/1_PgxV4pxlNTa0Ep9TDY_57oJfVtrofmu/edit#gid=1609043836): 

We have to decide whether to use a minimalistic circle or an svg with a bit more detail to include a wavy outline, some branches, etc. I would like to use more than a simple circle but without adding weight to our program with complex svg's. 

And some variation would be nice so that not all trees look the same. We could rotate the same design (unless it includes shadow) but it shouldn't be hard to add some variety to our options. 

Another idea would be to include an icon with each tree to differentiate it. For example a chesntut tree would have a chestnut icon, an apple tree an apple, an oak and acorn, and so on. We could include icons for nitrogen fixers or pollinizers too. I don't know that I like this idea but it is something to consider!

**Comment by berteh** [comment]
I would check out what's efficient in kivy, that already may handle wanted interactions and such.

I think I would create a [widget](https://kivy.org/doc/stable/api-kivy.uix.widget.html#kivy.uix.widget.Widget) for each plant. Widgets inherently have a [canvas](https://kivy.org/doc/stable/api-kivy.graphics.html#kivy.graphics.Canvas) on which to draw (for 1st proto) an inner circle (trunk) and outer circle (mature limit), and then add interactions on top.

That would be plenty cool for [0.4](https://github.com/jwnigel/permaculture/milestone/11)

We can later decide to replace the circles with characteristics-based images (type, size, color, family or even variety, with some fallback mechanism), eg for [1.3](https://github.com/jwnigel/permaculture/milestone/14) ... but feel free to move the milestone foward by changing its number if this is a priority for you.

**Comment by berteh** [comment]
In the meantime If you NEED to do it now then there are those assets that fit the need for basic shapes, and are easy to convert to SVG.
I can do it in a few minutes if you want: https://github.com/berteh/plantotech/tree/master/assets/img/silhouettes

... but I would (re)do it cleanly once the different types/value are established and you/we've decided on a general visual style. 
Would your student be interested to work on some more graphics aspects ?

**Comment by berteh** [comment]
and graphics brush palettes are usually a great source for such material as wel, such as [bushes](https://duckduckgo.com/?q=gimp+brush+bushes&atb=v347-1&ia=web), [tree leaves](https://duckduckgo.com/?q=gimp+brush+tree+leaves&atb=v347-1&ia=web) and much more, or replace "gimp" with "photoshop" to find more.

this one seems nice for instance: https://www.deviantart.com/redheadstock/art/Vegetation-Foliage-Textures-Photoshop-Brushes-534335625

please always *check usage conditions* before use, not to infringe copyright... since we will somehow be "repackaging" some creation into this app, that is often out of scope for the free licences. Most of the time, some acknowledgement +link in the credits page is enough for opensource material.

**Comment by jwnigel** [comment]
> I would check out what's efficient in kivy, that already may handle wanted interactions and such.
> 
> I think I would create a [widget](https://kivy.org/doc/stable/api-kivy.uix.widget.html#kivy.uix.widget.Widget) for each plant. Widgets inherently have a [canvas](https://kivy.org/doc/stable/api-kivy.graphics.html#kivy.graphics.Canvas) on which to draw (for 1st proto) an inner circle (trunk) and outer circle (mature limit), and then add interactions on top.

Yes, that's a good point. Let's begin by using two circles and maybe a transparent green fill. Then down the road we can improve the graphics. Very cool silohuettes you created, we will use those in the side view. 

My student isn't going to be creating the graphics himself--I didn't mean to say that. But he is helping me learn how to make them myself. Yesterday we spent an hour in Illustrator and afterwards when I tried to work by myself I became very frustrated. So I'll be glad to use circles for the moment.

**Comment by berteh** [comment]
The repository I mentioned is not mine, but a fork of another online plants database. still useable for a quick prototype.

That being said I'll gladly create the graphics you'd like. I'm fast enough for such things I think. Just need to discuss 

- the style, if you'd rather have something that looks hand-drawn, computer-generated, or photo-collage like.
- the items list : leaves/canopy, body/trunk, roots, soils types, flowers types (small/big)
- perspective : side / top / isomorphic

but again: I personnally see this (way) down the road, unless you plan to be done soon, then I get started working on this ;)

**Comment by jwnigel** [comment]
No, it's definitely something for down the road, when the structure is finished and we can begin to make things look nice!
